deps/config/migrations/000001_create_tables.up.sql

```diff
@@ -0,0 +1,51 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS secrets (
+    id VARCHAR (50) NOT NULL,
+    version VARCHAR (50) NOT NULL,
+    store_id VARCHAR (50) NOT NULL,
+    disabled BOOLEAN default false,
+    tags JSONB,
+    created_at TIMESTAMPTZ DEFAULT (now() at time zone 'utc') NOT NULL,
+    updated_at TIMESTAMPTZ DEFAULT (now() at time zone 'utc') NOT NULL,
+    deleted_at TIMESTAMPTZ,
+    PRIMARY KEY (id, version, store_id)
+);
+
+CREATE TABLE IF NOT EXISTS keys (
+    id VARCHAR (50) NOT NULL,
+    store_id VARCHAR (50) NOT NULL,
+    public_key BYTEA NOT NULL,
+    signing_algorithm VARCHAR (50) NOT NULL,
```

Comment:
There is not additional value in using VARCHARs onto TEXT, therefore I would rather replace all VARCHAR to TEXT to prevent us for doing wrong assumption about field sizes

Reply:
done